usm_rest_api/v1/urls.py

```diff
@@ -2,10 +2,16 @@
 from rest_framework import routers
 from .views import views as v1_views
 
+from djcelery import views as celery_views
+
 router = routers.DefaultRouter(trailing_slash=False)
 router.register(r'users', v1_views.UserViewSet)
 router.register(r'clusters', v1_views.ClusterViewSet)
 router.register(r'hosts', v1_views.HostViewSet)
+router.register(r'storage-devices', v1_views.StorageDeviceViewSet)
+router.register(r'discovered-hosts', v1_views.DiscoveredNodeViewSet)
+router.register(r'host-interfaces', v1_views.HostInterfaceViewSet)
+router.register(r'ceph/osds', v1_views.CephOSDViewSet)
```

Comment:
This seems to break with the hierarchy of the calamari API. Which is surprising and unexpected to me. Would you please move it to a compatible endpoint?